src/core_landice/Registry.xml

```diff
@@ -127,7 +127,11 @@
 	<nml_record name="calving" in_defaults="true">
 		<nml_option name="config_calving" type="character" default_value="none" units="unitless"
 		            description="Selection of the method for calving ice (as defined further below)."
-		            possible_values="'none', 'floating', 'topographic_threshold', 'thickness_threshold', 'eigencalving', 'mask'"
+		            possible_values="'none', 'floating', 'topographic_threshold', 'thickness_threshold', 'eigencalving', 'von_Mises_stress', 'mask'"
+		/>
+                <nml_option name="config_use_Albany_flowA_eqn" type="logical" default_value=".false." units="unitless"
```

Comment:
Let's change this to something like 'config_use_Albany_flowA_eqn_for_vM'.   Nobody like overly verbose option names, but unfortunately the way Fortran handles namelist options internally (and how they are written into the netCDF file metadata), there is no indication of which group an option belongs to.  So I think it could prevent confusion to somehow make it clear that this option is not a general thing but just applies to the von Mises stress calculation.

Reply:
Changed from config_use_Albany_flowA_eqn to config_use_Albany_flowA_eqn_for_vM